pkgs/tools/typesetting/tex/texpresso/tectonic.nix

```diff
@@ -0,0 +1,19 @@
+{ tectonic-unwrapped, fetchFromGitHub }:
+tectonic-unwrapped.override (old: {
```

Comment:
Can a simple overrideAttrs not be used here instead of overriding rustPlatform to inject the new src field?